golang/cosmos/app/app.go

```diff
@@ -365,7 +365,7 @@ func NewAgoricApp(
 
 	// This function is tricky to get right, so we build it ourselves.
 	callToController := func(ctx sdk.Context, str string) (string, error) {
-		if vm.IsSimulation(ctx) {
+		if sendToController == nil || vm.IsSimulation(ctx) {
```

Comment:
Add to comment below: "is being simulated ... or we're deliberately running without SwingSet".

Reply:
I removed this loosey-goosey `nil` check.  Now the mechanism is different.

Reply:
> OMG a `==`. Don't our lint rules prohibit it?

Yes they do, but this is Golang, not JS.